routes/apiProductRoutes.js

```diff
@@ -1,84 +1,87 @@
-// // *********************************************************************************
-// // api-routes.js - this file offers a set of routes for displaying and saving data to the db
-// // *********************************************************************************
 
-// // Dependencies
-// // =============================================================
+const db = require("../models");
 
-// // Requiring our Todo model
-// var db = require("../models");
 
-// // Routes
-// // =============================================================
-// module.exports = function(app) {
+// Routes
+// =============================================================
+module.exports = function(app) {
 
-//   // GET route for getting all of the posts
-//   app.get("/api/posts/", function(req, res) {
-//     db.Post.findAll({})
-//       .then(function(dbPost) {
-//         res.json(dbPost);
-//       });
-//   });
+  // GET route for getting all of the posts
+  app.get("/api/posts/", function(req, res) {
+    db.Post.findAll({})
```

Comment:
@Marifer-Enriquez There is no `Post` model, I think we should be referencing the `Product` model. I guess we should update that reference.

Reply:
done!